unblob/extractor.py

```diff
@@ -62,7 +62,13 @@ def fix_symlink(path: Path, outdir: Path, task_result: TaskResult) -> Path:
         path.unlink()
         return path
 
-    target = Path(os.readlink(path))
+    raw_target = os.readlink(path)
+    if not raw_target:
+        logger.error("Symlink with empty target, removing.")
+        path.unlink()
+        return path
+
+    target = Path(raw_target)
```

Comment:
Would worth a new case in `test_fix_symlink`

Reply:
It's such an edge case that I can't reproduce it through Python code.

call | OS | exception
----|-----|------------
`Path('test').symlink_to('')` | Linux | `FileNotFoundError`
`Path('test').symlink_to('')` | Darwin| `OSError`
`os.symlink('test', '')` | Linux| `FileNotFoundError`
`os.symlink('test', '')` | Darwin | `FileNotFoundError`

It's not worth implementing a test based on a file since it's already done by the `f_badsymlink.img` file in integration tests. If we have a regression, the extfs integration test will fail on Darwin.